src/compiler/crystal/macros.cr

```diff
@@ -67,6 +67,17 @@ module Crystal::Macros
   def raise(message) : NoReturn
   end
 
+  # Reads a file: if it exists, returns a StringLiteral with its contents;
+  # otherwise `nil` is returned.
+  #
+  # To read a file relative to where the macro is defined, use:
```

Comment:
Maybe add that paths are relative to the current working directory. That's already implied, but can't hurt to make it explicit.